Assets/Scripts/Domain/UseCase/MusicUseCase.cs

```diff
@@ -71,6 +72,7 @@ public void Play(TEnum key, bool loop = true, bool keepIfSame = true)
 
         public void Stop()
         {
+            MusicModel.AudioClip.Value = default(AudioClip);
             MusicModel.MusicPlayer.Stop();
```

Comment:
```suggestion
            MusicModel.MusicPlayer.Stop();
            MusicModel.AudioClip.Value = default(AudioClip);
```

大丈夫だとは思いますが、null を設定するのは音を止めてからの方が良いかも知れません。
（ノイズとかがちょっとだけ心配です。）

Reply:
たしかにその順番のほうがいいですね！